content/en/docs/instrumentation/python/automatic/agent-config.md

```diff
@@ -26,6 +28,12 @@ opentelemetry-instrument \
 
 Here's an explanation of what each configuration does:
 
+- `distro` specifies which distro should be loaded. Argument should be the entry
+  point name of the desired distro. If only a single distro exists, this
+  argument may be left out.
+- `configurator` specifies which configurator should be loaded. Argument should
+  be the entry point name of the desired configurator. If only a single
+  configurator exists, this argument may be left out.
```

Comment:
+1 even as someone involved I don't remember what distro vs configurator is. Would be nice to have something explaining what they are.

Reply:
We do have documentation (at least for distros) [here](https://opentelemetry.io/docs/instrumentation/python/distro/). I'm not sure this is the place where we should document what a distro or a configurator is. We don't do any explanation for `--traces_exporter` or `--metrics_exporter` either.

Maybe just a link to the general documentation would be enough here?

Reply:
I always found the distros page you linked confusing because it replicates a lot of what is written in the automatic documentation, so we need to merge them eventually. Also, Python is the only language as far as I am aware that has this understanding of a distribution. 

I have no issue with `distro` and `configurator` being documented as outlined here, but what we need is examples what options I have as an enduser, if there are currently only 1 option and they will come later, then let's postpone this for now and add it when people have a choise.

> We don't do any explanation for --traces_exporter or --metrics_exporter either.

We could have them.